bot/cogs/games/hangman.py

```diff
@@ -0,0 +1,193 @@
+import random
+import re
+import typing as t
+from collections import defaultdict
+from os import path
+
+from discord import Color, Embed, Guild, Member, Message, TextChannel
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot import constants
+
+DIR_PATH = path.dirname(__file__)
```

Comment:
What's this, i don't know about this.

Reply:
It's just a DIR_PATH constant, so I don't need to use `path.dirname(__file__)` to get the name of the directory `hangman.py` is in, in this case, it will return `bot/cogs/games`, if it were moved to some other directory, it would change to that.